torch/fx/experimental/symbolic_shapes.py

```diff
@@ -2087,6 +2164,16 @@ def simplify(self, expr: "sympy.Expr") -> "sympy.Expr":
             # divisions simplified away
             if new_pows.issubset(pows) and new_rationals.issubset(rationals):
                 expr = new_expr
+        if isinstance(expr, sympy.Rel):
+            lhs_const = get_added_const(expr.lhs)
+            rhs_const = get_added_const(expr.rhs)
+            if (
+                    lhs_const is not None
+                    and rhs_const is not None
+                    and lhs_const != 0
```

Comment:
I think we should do this transformation even when the `lhs_const` is zero. In that case, you should just unwrap the inner expression from the lhs and remove the zero.

Reply:
I would say we could remove it, since `sympy` simplifies it automatically. But, what if we have something like `0 < expr`?

Reply:
Ok, so I?m a bit lost here. Do we want to do this simplification in other cases as well, or just in the setting of this PR. In the setting of this PR, we cannot have that situation, as we know by construction that we have at least one variable on the LHS.

Reply:
On second thoughts, I don't think it makes any significant difference given that `lhs_const == 0`